aicsimageio/readers/bfio_reader.py

```diff
@@ -0,0 +1,285 @@
+
+        return self._general_data_array_constructor(
+            da.from_array(self._rdr, chunks=(1024, 1024) + (1,) * (self._rdr.ndim - 2)),
+            self._tiff_tags(),
+        )
+
+    def _tiff_tags(self) -> Optional[Dict[str, str]]:
+
+        tiff_tags: Optional[Dict[str, str]] = None
+        if self.backend == "python":
+            # Create a copy since TiffTags are not serializable
+            tiff_tags = {
+                code: tag.value
+                for code, tag in self._rdr._backend._rdr.pages[0].tags.items()
+            }
+
+        return tiff_tags
+
+    def _read_immediate(self) -> xr.DataArray:
+        return self._general_data_array_constructor(
+            self._rdr[:],
+            self._tiff_tags(),
+        )
+
+    @property
+    def ome_metadata(self) -> OME:
+        return self._rdr.metadata
+
+
+class OmeTiledTiffReader(BfioReader):
+    """
+    Wrapper around bfio.BioReader(backend="python").
+
+    The OmeTiledTiffReader is an optimized TIFF reader written in pure Python, built on
+    top of tifffile. This reader is optimized for speed and scalability, but will only
+    read TIFF files that meet the following requirements:
+
+    1. TileWidth and TileHeight tags must both be set to 1024
+    2. The Description tag must contain OMEXML
+    3. The OMEXML channel ordering must be set to XYZCT
+    4. Channels cannot be interleaved, meaning individual channels must be planes.
+
+    The advantage of the reader for files that meet these requirements are improvements
+    in reading speed, especially when accessing data using dask.
+
+    This TIFF reader will only read the first image and pyramid layer. If pyramid layers
+    or images beyond the first image in the file need to be read, use the OmeTiffReader.
+
+    Parameters
+    ----------
+    image: types.PathLike
+        Path to image file.
+    chunk_dims: List[str]
+        Which dimensions to create chunks for.
+        Default: DEFAULT_CHUNK_DIMS
+        Note: Dimensions.SpatialY, and Dimensions.SpatialX will always be added to the
+        list if not present during dask array construction.
+    out_order: List[str]
+        The output dimension ordering.
+        Default: DEFAULT_DIMENSION_ORDER
+
+    Notes
+    -----
+    If the OME metadata in your file isn't OME schema compliant or does not validate
+    this will fail to read your file and raise an exception.
+
+    If the OME metadata in your file doesn't use the latest OME schema (2016-06),
+    this reader will make a request to the referenced remote OME schema to validate.
+    """
+
+    backend: str = "python"
+
+    @staticmethod
+    def _is_supported_image(fs: AbstractFileSystem, path: str, **kwargs: Any) -> bool:
+        try:
+            if not isinstance(fs, LocalFileSystem):
+                return False
+
+            with BioReader(path, backend="python") as br:
+
+                # Fail fast if multi-image file
+                assert len(br.metadata.images) == 1
```

Comment:
Nitpick, could you make this:

```python
if len(br.metadata.images) > 1:
    raise UnsupportedFileFormatError("This file contains multiple scenes, recommended to use either the base BioReader if you want to use bfio, or the base TiffReader for base aicsimageio as this reader only supports reading the first scene.")
```

Not the happiest with that error message but I think you get the idea. I just want a bit more documentation as to why the reader is rejecting the image.